platform/osx/detect.py

```diff
@@ -57,22 +57,15 @@ def configure(env):
 
     ## Architecture
 
-    is64 = sys.maxsize > 2**32
-    if (env["bits"] == "default"):
-        env["bits"] = "64" if is64 else "32"
+    # Mac OS X no longer runs on 32-bit since 10.7 which is unsupported since 2014
+    # As such, we only support 64-bit
+    env["bits"] == "64"
```

Comment:
@akien-mga: Should this be `==` or `=`?

Reply:
THis should be `=` indeed, thanks. I'll fix that.